dart/constraint/BoxedLcpConstraintSolver.hpp

```diff
@@ -116,6 +116,15 @@ class BoxedLcpConstraintSolver : public ConstraintSolver
   std::vector<s_t*> solveConstrainedGroup(
       ConstrainedGroup& group, simulation::World* world) override;
 
+  /// Build the inputs to the LCP from the constraint group.
+  bool buildLcpInputs(ConstrainedGroup& group);
+
+  /// Solve LCP from inputs.
```

Comment:
I know I don't always write good comments, but this is one of those "this is a stop sign" comments. Maybe "Setup and solve an LCP to enforce the constraints on the ConstrainedGroup"?